palisade-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/palisade/kafka/KafkaContractTest.java

```diff
@@ -207,12 +221,15 @@ void testRestEndpointError() {
         // Then - the results are as expected
         // The request was written with the correct header
         assertThat(results)
+                .as("Check there is only 1 result from the error probe")
                 .hasSize(1)
                 .allSatisfy(result -> {
                     assertThat(result.value()).usingRecursiveComparison()
                             .ignoringFieldsOfTypes(Throwable.class).ignoringFields("timestamp")
+                            .as("Recursively compare the AuditErrorMessage object, ignoring the Throwable and TimeStamp values")
                             .isEqualTo(ContractTestData.ERROR_OBJ);
                     assertThat(result.headers().lastHeader(Token.HEADER).value())
```

Comment:
I've been having an issue getting this to work when returning the message within the error, it's `null` for some reason. If you really want to include an assertion for this then it might be worth investigating in more depth as part of a ticket.

Reply:
Added extra assertion on error class.